actionpack/lib/action_controller/metal/head.rb

```diff
@@ -50,16 +50,16 @@ def head(status, options = {})
     end
 
     private
-    # :nodoc:
-    def include_content?(status)
-      case status
-      when 100..199
-        false
-      when 204, 205, 304
-        false
-      else
-        true
```

Comment:
@vipulnsward Can you keep indentation as it is and move `nodoc` at correct place?